<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgMthxDialogUtil</class>
 <widget class="QDialog" name="dlgMthxDialogUtil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MthxDatabaseUtil</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout" stretch="1,1,1,1,100,1,1">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Mthx Database Utility</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,8,2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Function</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbDBFunc"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelDBFunc">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Source1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSrc1"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelSrc1">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Source2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSrc2"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelSrc2">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Target</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtSrc3"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelTar">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tabViewLog"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnEnd">
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
